src/features/expand-collapse-outdated-comments.js

```diff
@@ -0,0 +1,40 @@
+import select from 'select-dom';
+import {isMac, metaKey} from '../libs/utils';
+
+const keyName = isMac ? 'Alt' : 'Ctrl';
+
+function addTooltip(button) {
+	button.setAttribute('aria-label', keyName + ' + click to expand/collapse all outdated comments');
+	button.classList.add('tooltipped', 'tooltipped-n');
```

Comment:
I think you’ll also need the `rgh-tooltip` here otherwise they’re hidden

Reply:
I saw you added more selectors to the exclusion CSS. This class here would remove the need to make that selector longer.